docs/semgrep-supply-chain/getting-started.md

```diff
@@ -85,6 +85,34 @@ You can configure your CI/CD system to trigger a Semgrep Supply Chain scan whene
  </tr>
 </table>
 
+## Scan a project without lockfiles (beta)
+
+:::info
+Semgrep does not support managed scans of projects without lockfiles.
```

Comment:
Are we saying that SMS doesn't scan projects without lockfiles at all? What happens? The SSC scan fails? Is there a way for the user to receive feedback? I feel like this is going to trip up users.

Reply:
@bkettle can you please clarify? ☝️

Reply:
Ah, good point, I could see how this could be confusing. All that happens is that we aren't able to determine the dependencies when there is no lockfile, so the supply chain scan will not find any findings (but nothing will fail).